### PR TITLE
Prepare release 0.1.2

### PR DESCRIPTION
**Pull request by Zifzaf**
- Document changes in the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.2] - 2020-08-24
+
+ * `[fixed]` Fix overflow bug with high flow rates (#19)
```
Comment (abrauchli) [inline CHANGELOG.md:10]: I think it's #20

Comment (abrauchli) [reply CHANGELOG.md:10]: should we indicate at what flow rate the overflow would have happened? @rnestler

**Review by abrauchli — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rnestler) [reply CHANGELOG.md:10]: > should we indicate at what flow rate the overflow would have happened?

Yeah, maybe add something like "... with flow rates higher than $FOO"

**Comment by Zifzaf** [comment]
Changed to (#20). Raphi and me also decided that there shouldn't be a rate specified, since the rate depends on the offset of the sensor.

**Review by abrauchli — APPROVED** [review]

**Review by abrauchli — CHANGES_REQUESTED** [review]
ah, just saw the merge in the history, can you rebase instead?

**Review by abrauchli — APPROVED** [review]

**Pull request merged**